### PR TITLE
Update prerequisites section for additional requirement

### PR DESCRIPTION
**Pull request by vasukulkarni**
Signed-off-by: Vasu Kulkarni <vasu@redhat.com>

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
 
 1. Python 3.7
 2. AWS Account credentials when testing with AWS platforms
+   check default section in ~/.aws/credentials for access/secret key.
```
Comment (mbukatov) [inline docs/getting_started.md]: Maybe you want to link to https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html

File: docs/getting_started.md [hunk 2]
```diff
@@ -5,6 +5,9 @@
...
+3. OC client binary is installed on your localhost in /usr/local/bin or
+   /usr/bin(running oc version on terminal should display version > 3.11).
```
Comment (mbukatov) [inline docs/getting_started.md]: You mean to have `oc` placed somewhere in `PATH`. Also you may want to link to https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/

**Comment by vasukulkarni** [comment]
This should be squashed and merged. Thanks

**Comment by vasukulkarni** [comment]
@clacroix12 this had got the required approvals before, please check the final review commit which I had missed before and squash/merge. Thanks

**Pull request merged**